### DISCUSSION TITLE
Showing a list of running automations on restart screen

### DISCUSSION THREAD

**Discussion by hpuac** [home-assistant/frontend#21103] [Other feature requests] [answered]
Hey folks, what do you think about a feature that lists all automations that are currently being executed on the screen where you can restart HA?
By having that information I could choose to not restart HA yet until the automations are finished.
This would have avoided me coming home to half of my blinds still being open as I restarted the instance remotely while the closing automation was being executed 😁 
Let me know what you think, maybe it's also useful for others.
Cheers!

**Comment by N1ckn4m3sN3t** [comment]
Bump

This is an old question, but a very useful one. I'd also like to see this feature on the updates page for updates that require a restart.

Or maybe someone knows of an add-on that gives me an overview of running automations?

Cheers

**Comment by karwosts** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/53523
